pandas/tests/groupby/test_grouping.py

```diff
@@ -282,6 +282,16 @@ def test_groupby_categorical_index_and_columns(self, observed):
         expected = DataFrame(data=expected_data.T, index=expected_columns)
         assert_frame_equal(result, expected)
 
+    def test_groupby_extension_array(self):
```

Comment:
so move this to pandas/tests/arrays/test_integer. There is a test there that might actually need to change as well. We ideally would also have a very generic version of this in  pandas/tests/extension/base/groupby.py